### PR TITLE
test: add unit test case for static_locator.go

### PR DESCRIPTION
**Pull request by fenggw-fnst**
Signed-off-by: Guangwen Feng <fenggw-fnst@cn.fujitsu.com>

### Ⅰ. Describe what this PR did
Add unit test case for static_locator.go

### Ⅱ. Does this pull request fix one issue?

NONE

### CODE REVIEW

File: dfget/locator/static_locator_test.go [hunk 1]
```diff
@@ -170,6 +170,24 @@ func (s *StaticLocatorTestSuite) Test_Refresh(c *check.C) {
 	c.Assert(l.load(), check.Equals, -1)
 }
 
+func (s *StaticLocatorTestSuite) Test_String(c *check.C) {
+	cases := []struct {
+		locator  *StaticLocator
+		expected string
```
Comment (zhouhaibing089) [inline dfget/locator/static_locator_test.go:176]: I would suggest defining this case as:

```golang
cases := []struct {
    group      string
    locator    *StaticLocator
    expected   string
    increments int
}
```

File: dfget/locator/static_locator_test.go [hunk 2]
```diff
@@ -170,6 +170,24 @@ func (s *StaticLocatorTestSuite) Test_Refresh(c *check.C) {
...
+	}{
+		{createLocator("a:80=1"), "empty"},
+		{createLocator("a:80=1"), "empty"},
+		{createLocator("a:80=1"), "test-group:[a:80=1]"},
+	}
+	cases[0].locator.Group = nil
+	cases[1].locator.inc()
+	cases[1].locator.inc()
```
Comment (zhouhaibing089) [inline dfget/locator/static_locator_test.go:184]: These extra setups bothered me. (like what if the order changed in future?)

File: dfget/locator/static_locator_test.go [hunk 3]
```diff
@@ -170,6 +170,24 @@ func (s *StaticLocatorTestSuite) Test_Refresh(c *check.C) {
 	c.Assert(l.load(), check.Equals, -1)
 }
 
+func (s *StaticLocatorTestSuite) Test_String(c *check.C) {
+	cases := []struct {
+		locator  *StaticLocator
+		expected string
+	}{
+		{createLocator("a:80=1"), "empty"},
```
Comment (zhouhaibing089) [inline dfget/locator/static_locator_test.go:178]: nit: I'd also suggest using named fields:

```golang
{
    locator:  createLocator("a:80-1"),
    expected: "empty",
},
```

**Comment by fenggw-fnst** [comment]
@zhouhaibing089 Thanks for the review, done, PTAL.

**Comment by zhouhaibing089** [comment]
LGTM

**Pull request merged**